content/docs/user-guide/components/reference/gradients/gradient-baker.md

```diff
@@ -0,0 +1,56 @@
+---
+linktitle: Gradient Baker
+title: Gradient Baker Component
+description: Use the Gradient Baker component to bake a complex node graph of gradients down to a single static image in Open 3D Engine (O3DE).
+---
+
+Use the **Gradient Baker** component to bake a complex node graph of gradients down to a single static image. You can then use a single [Image Gradient](/docs/user-guide/components/reference/gradients/image-gradient/) at runtime, instead of having the entire node graph computed every time, to optimize performance.
+
+For example, take this complex gradient chain with a generator and multiple gradient modifiers.
+
+![Complex chain without gradient baker](/images/user-guide/components/reference/gradients/gradient-baker-chain-without-baker.png)
+
+You can create a **Gradient Baker** and connect its input gradient to the output of the gradient chain you'd like to bake. The **Gradient Baker** also needs an input bounds for where to sample the data. With both of those connected, the graph would look like this.
+
+![Complex chain with gradient baker](/images/user-guide/components/reference/gradients/gradient-baker-chain-with-baker.png)
+
+Clicking the **Bake image** button on the **Gradient Baker** component will then bake out a single static image that can be used in a single [Image Gradient](/docs/user-guide/components/reference/gradients/image-gradient/), which reduces the entire complex graph to only a single component.
+
+![Complex chain with gradient baker](/images/user-guide/components/reference/gradients/gradient-baker-single-image-gradient.png)
+
+The final optimization step is to select all of the gradients in your chain and mark them as **Editor only**. This will prevent them from being loaded/computed at runtime. You can optionally mark the **Gradient Baker** as **Editor only** as well, but it will have no effect because there is no runtime component for it anyways.
```

Comment:
The "anyways" at the end can be removed.

Reply:
Will do, thanks!